nanobind_stubgen/NanobindStubsGenerator.py

```diff
@@ -351,7 +351,7 @@ def _analyse_module(self, module: Any, stub_entry: StubEntry) -> StubModule:
             if inspect.isclass(obj):
                 if type(obj).__name__ == "nb_type":
```

Comment:
FYI this is no longer correct either -- there is a different nanobind metatype these days per amount of supplementary data that's being stored in the type object. Most nanobind classes have `type(obj).__name__ == "nb_type_0"` but some may have other numbers. The best discriminant is `type(type(obj)).__name__ == "nb_meta"`. And note that enums will satisfy that too, so check for them inside the if for types.

Reply:
Thanks for the info & hint. I will have a look at it as soon as possible. Is there an example library to test it, using nanobind >= 1.3.0?

Reply:
Sorry, I don't know anything about the universe of publicly available nanobind libraries; I use it for internal stuff at my day job. If I were maintaining a package like this I would probably do first-line testing of it with my own 'sample' nanobind extension rather than an external one.

Reply:
@cansik our next version of [Lagrange](https://github.com/adobe/lagrange) (v6.18.0) will update to Nanobind 1.8.0. It's not our yet (see [PR](https://github.com/adobe/lagrange/pull/21)) since we need to sync some missing unit-test data. The PyPi package is called [lagrange-open](https://pypi.org/project/lagrange-open/). I can ping you when we upload v6.18.0 to PyPi.

Reply:
@oremanj Yes that would be definitely better, however since I am not using nanobind myself (except the pre-built nanogui), I did not create such a sample-lib yet.

@jdumas Thanks, will have a look.

Reply:
@cansik we just published [lagrange-open](https://pypi.org/project/lagrange-open/) v6.18.0 to PyPi. It's using Nanobind 1.8.0 to generate Python bindings, so you can inspect the generated objects if you want.

May I suggest that we address the points raised by @oremanj about `nb_type` and `nb_enum` in a separate PR?